packages/react/src/stories/ic-text-field.stories.mdx

```diff
@@ -482,3 +482,207 @@ export const Uncontrolled = () => {
     ></IcTextField>
   </Story>
 </Canvas>
+
+### Playground
+
+export const defaultArgs = {
+  autocapitalize: "off",
+  autocomplete: "off",
+  autocorrect: "off",
+  autoFocus: false,
+  disabled: false,
+  fullWidth: false,
+  helperText: "",
+  hideLabel: false,
+  inputId: "ic-text-field-input-0",
+  inputmode: "text",
+  label: "Text Field",
+  maxCharacters: 0,
+  maxLength: 0,
+  minCharacters: 0,
+  name: "0",
+  placeholder: "",
+  readonly: false,
+  required: false,
+  resize: false,
+  rows: 1,
+  size: "default",
+  spellcheck: false,
+  type: "text",
+  validationInline: false,
+  validationStatus: "",
+  validationText: "",
+  debounce: 0
+};
+
+<Canvas>
+  <Story
+    args={defaultArgs}
+    argTypes={{
+      autocapitalize: {
```

Comment:
Not sure if I've misunderstood what this prop should do, it's not working as I expect based off the prop description: _"The automatic capitalisation of the text value as it is entered/edited by the user. Available options: "off", "none", "on", "sentences", "words", "characters"."_
<img width="1195" alt="Screenshot 2024-06-19 at 13 51 51" src="https://github.com/mi6/ic-ui-kit/assets/113986285/c42c0e4a-2568-4df7-b1cb-8c9ac4bfbb4b">